citestwheel/action.yml

```diff
@@ -0,0 +1,9 @@
+name: citestwheel for manylinux
+description: 'Runs wheel GPU tests for manylinux wheels'
+
+runs:
```

Comment:
Originally wanted citestwheel.sh here for quick iteration
When the test script is 'stable', it can be baked into a container.